addons/stock/models/stock_move.py

```diff
@@ -1040,6 +1040,8 @@ def _update_reserved_quantity(self, need, available_quantity, location_id, lot_i
             to_update = self.move_line_ids.filtered(lambda ml: ml._reservation_is_updatable(quantity, reserved_quant))
             if to_update:
                 to_update[0].with_context(bypass_reservation_update=True).product_uom_qty += self.product_id.uom_id._compute_quantity(quantity, to_update[0].product_uom_id, rounding_method='HALF-UP')
+                if to_update[0].result_package_id:
+                    to_update[0].qty_done = to_update[0].product_uom_qty
```

Comment:
what? why?

Reply:
I consider this case, If 80 as the initial quantity and 60 as a reserved quantity and if we put in the pack then done quantity is 60 after that when 20 quantity is available and then we validate the transfer then backorder is created.
but this solution is not valid so, I will revert this.